Library/Homebrew/dev-cmd/install-bundler-gems.rb

```diff
@@ -13,7 +13,8 @@ def install_bundler_gems_args
         Install Homebrew's Bundler gems.
       EOS
       comma_array "--groups",
-                  description: "Installs the specified comma-separated list of gem groups (default: last used)."
+                  description: "Installs the specified comma-separated list of gem groups (default: last used). " \
+                               "Prepend with `+` to install these groups in addition to what is already installed."
```

Comment:
Mmm, this feels a little weird to me. What about having `--add-groups` or something instead?